busco/busco test can fail

### Have you checked the docs?

- [X] [nf-core website: troubleshooting](https://nf-co.re/usage/troubleshooting)
- [X] [nf-core modules documentation](https://nf-co.re/docs/contributing/modules)

### Description of the bug

In the `test_busco_genome_multi_fasta` of `busco/busco`, the output file `test-bacteria_odb10-busco.batch_summary.txt` seems to be outputting the input files in a random order, breaking the MD5 check

cc @mahesh-panchal 

### Command used and terminal output

```console
$ env TMPDIR=~/nfs/scratch123/nextflow PROFILE=singularity pytest --symlink --git-aware --keep-workflow-wd-on-fail --tag busco
====================================================================== test session starts ======================================================================
platform linux -- Python 3.9.5, pytest-7.0.1, pluggy-1.0.0
rootdir: /nfs/users/nfs_m/mm49/workspace/tol-it/nextflow/modules, configfile: pytest.ini
plugins: workflow-1.6.0
collecting ... 
collected 999 items                                                                                                                                             

busco test_busco_genome_single_fasta:
        command:   nextflow run tests/modules/busco -entry test_busco_genome_single_fasta -c tests/config/nextflow.config
        directory: /nfs/users/nfs_m/mm49/nfs/scratch123/nextflow/pytest_workflow_iw3v1jti/busco_test_busco_genome_single_fasta
        stdout:    /nfs/users/nfs_m/mm49/nfs/scratch123/nextflow/pytest_workflow_iw3v1jti/busco_test_busco_genome_single_fasta/log.out
        stderr:    /nfs/users/nfs_m/mm49/nfs/scratch123/nextflow/pytest_workflow_iw3v1jti/busco_test_busco_genome_single_fasta/log.err
'busco test_busco_genome_single_fasta' done.

busco test_busco_genome_multi_fasta:
        command:   nextflow run tests/modules/busco -entry test_busco_genome_multi_fasta -c tests/config/nextflow.config
        directory: /nfs/users/nfs_m/mm49/nfs/scratch123/nextflow/pytest_workflow_iw3v1jti/busco_test_busco_genome_multi_fasta
        stdout:    /nfs/users/nfs_m/mm49/nfs/scratch123/nextflow/pytest_workflow_iw3v1jti/busco_test_busco_genome_multi_fasta/log.out
        stderr:    /nfs/users/nfs_m/mm49/nfs/scratch123/nextflow/pytest_workflow_iw3v1jti/busco_test_busco_genome_multi_fasta/log.err
'busco test_busco_genome_multi_fasta' done.
busco test_busco_eukaryote_metaeuk:
        command:   nextflow run tests/modules/busco -entry test_busco_eukaryote_metaeuk -c tests/config/nextflow.config
        directory: /nfs/users/nfs_m/mm49/nfs/scratch123/nextflow/pytest_workflow_iw3v1jti/busco_test_busco_eukaryote_metaeuk
        stdout:    /nfs/users/nfs_m/mm49/nfs/scratch123/nextflow/pytest_workflow_iw3v1jti/busco_test_busco_eukaryote_metaeuk/log.out
        stderr:    /nfs/users/nfs_m/mm49/nfs/scratch123/nextflow/pytest_workflow_iw3v1jti/busco_test_busco_eukaryote_metaeuk/log.err
'busco test_busco_eukaryote_metaeuk' done.

busco test_busco_eukaryote_augustus:
        command:   nextflow run tests/modules/busco -entry test_busco_eukaryote_augustus -c tests/config/nextflow.config
        directory: /nfs/users/nfs_m/mm49/nfs/scratch123/nextflow/pytest_workflow_iw3v1jti/busco_test_busco_eukaryote_augustus
        stdout:    /nfs/users/nfs_m/mm49/nfs/scratch123/nextflow/pytest_workflow_iw3v1jti/busco_test_busco_eukaryote_augustus/log.out
        stderr:    /nfs/users/nfs_m/mm49/nfs/scratch123/nextflow/pytest_workflow_iw3v1jti/busco_test_busco_eukaryote_augustus/log.err
'busco test_busco_eukaryote_augustus' done.

busco test_busco_protein:
        command:   nextflow run tests/modules/busco -entry test_busco_protein -c tests/config/nextflow.config
        directory: /nfs/users/nfs_m/mm49/nfs/scratch123/nextflow/pytest_workflow_iw3v1jti/busco_test_busco_protein
        stdout:    /nfs/users/nfs_m/mm49/nfs/scratch123/nextflow/pytest_workflow_iw3v1jti/busco_test_busco_protein/log.out
        stderr:    /nfs/users/nfs_m/mm49/nfs/scratch123/nextflow/pytest_workflow_iw3v1jti/busco_test_busco_protein/log.err
'busco test_busco_protein' done.

busco test_busco_transcriptome:
        command:   nextflow run tests/modules/busco -entry test_busco_transcriptome -c tests/config/nextflow.config
        directory: /nfs/users/nfs_m/mm49/nfs/scratch123/nextflow/pytest_workflow_iw3v1jti/busco_test_busco_transcriptome
        stdout:    /nfs/users/nfs_m/mm49/nfs/scratch123/nextflow/pytest_workflow_iw3v1jti/busco_test_busco_transcriptome/log.out
        stderr:    /nfs/users/nfs_m/mm49/nfs/scratch123/nextflow/pytest_workflow_iw3v1jti/busco_test_busco_transcriptome/log.err
'busco test_busco_transcriptome' done.

tests/test_versions_yml.py ssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss [ 12%]
sssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss [ 27%]
sssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss [ 43%]
sssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss [ 58%]
sssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss [ 73%]
ssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss......sssssssssssssssssssssssssssssssssssssssss         [ 88%]
tests/modules/busco/test.yml ........................................................F...........................................................         [100%]One or more tests failed. Keeping temporary directories and logs. Use '--kwd' or '--keep-workflow-wd' to disable this behaviour.


=========================================================================== FAILURES ============================================================================
_________________________________________________________________________ test session __________________________________________________________________________
Observed md5sum '3bb11d77e139f08a0d90fbab0a3b30f2' not equal to expected md5sum '5360dfe83bec1f5741ee115e53e6b517' for file '/nfs/users/nfs_m/mm49/nfs/scratch123/nextflow/pytest_workflow_iw3v1jti/busco_test_busco_genome_multi_fasta/output/busco/test-bacteria_odb10-busco.batch_summary.txt'
==================================================================== short test summary info ====================================================================
FAILED tests/modules/busco/test.yml::busco test_busco_genome_multi_fasta::output/busco/test-bacteria_odb10-busco.batch_summary.txt::md5sum
==================================================== 1 failed, 121 passed, 877 skipped in 429.44s (0:07:09) =====================================================
```


### Relevant files

The `busco_test_busco_genome_multi_fasta/output/busco/test-bacteria_odb10-busco.batch_summary.txt` I got is:
```
Input_file      Dataset Complete        Single  Duplicated      Fragmented      Missing n_markers
genome.fasta    bacteria_odb10  46.0    46.0    0.0     12.1    41.9    124
genome.fna      bacteria_odb10  97.6    95.2    2.4     2.4     0.0     124
```
MD5: `3bb11d77e139f08a0d90fbab0a3b30f2`

You'll notice that if you reorder the lines (1, 3, 2), the MD5 becomes the correct one: `5360dfe83bec1f5741ee115e53e6b517`


### System information

The machine is on our (on-premises) HPC:
```
$ nextflow -version                                                                                        
      N E X T F L O W
      version 22.04.0 build 5697
      created 23-04-2022 18:00 UTC (19:00 BST)
      cite doi:10.1038/nbt.3820
      http://nextflow.io

$ singularity --version
singularity-ce version 3.9.0

$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 18.04.6 LTS
Release:        18.04
Codename:       bionic
```

modules commit ID: `5affd97c`

--- a/tests/modules/busco/test.yml
+++ b/tests/modules/busco/test.yml
@@ -72,7 +72,18 @@
         - "Missing BUSCOs"
         - "Dependencies and versions"
     - path: output/busco/test-bacteria_odb10-busco.batch_summary.txt
-      md5sum: 5360dfe83bec1f5741ee115e53e6b517
+      contains:
+        - "Input_file"
+        - "Dataset"
+        - "Complete"
+        - "Single"
+        - "Duplicated"
+        - "Fragmented"
+        - "Missing"
+        - "n_markers"
+        - "genome.fasta"
+        - "genome.fna"
+        - "bacteria_odb10"
     - path: output/busco/versions.yml
       md5sum: 9a959eb0a1f765777dff1ea2f5c139c0
 

